src/runtime/components/forms/Range.vue

```diff
@@ -89,7 +89,16 @@ export default defineComponent({
 
     const { emitFormChange, formGroup } = useFormGroup()
     const color = computed(() => formGroup?.error?.value ? 'red' : props.color)
-    const size = computed(() => formGroup?.size?.value ?? props.size)
+    const size = computed(() => {
+      const size = formGroup?.size?.value ?? props.size
+      if (!ui.value.size[size.value] ||
```

Comment:
I think we don't need to check too many cause that issue happens because the `range` component do not support the `xl` size from `formGroup`.

Reply:
agree, is it better now?

Reply:
I would prefer return to `props.size`, that might be better for users want to define `range size` themselves, and it has default value already.
Return to default will overwrite the `size` passed by `range` if `range` not support `formGroup size`.

Reply:
done